examples/v3/multipart/consumer.spec.ts

```diff
@@ -0,0 +1,231 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import {
+  SpecificationVersion,
+  PactV3,
+  MatchersV3,
+} from '@pact-foundation/pact';
+import FormData from 'form-data';
+import axios from 'axios';
+
+chai.use(chaiAsPromised);
+
+const { expect } = chai;
+
+describe('Pact Consumer Test', () => {
+  // Initialize a new Pact instance with consumer and provider names
+  // Using V3 specification to support multipart form data
+  const pact = new PactV3({
+    consumer: 'myconsumer',
+    provider: 'myprovider',
+    spec: SpecificationVersion.SPECIFICATION_VERSION_V3,
+    logLevel: 'trace',
+  });
+
+  it('creates a pact to verify with a text field in form', async () => {
+    // Create a multipart form data object with a 'name' field
+    const formData = new FormData();
+    formData.append('name', 'John Doe');
+
+    // Extract the headers that include the Content-Type with the boundary parameter
+    // This boundary is crucial for multipart form data parsing
+    const headers = formData.getHeaders();
+
+    // Convert the form data to a string representation for the request body
+    const form = formData.getBuffer().toString();
+
+    await pact
+      .addInteraction({
+        uponReceiving: 'a request for a foo',
+        withRequest: {
+          method: 'POST',
+          path: '/test',
+          body: form, // The multipart form data as a string
+          headers, // Headers including Content-Type with boundary
+        },
+        willRespondWith: {
+          status: 201,
+          body: {
```

Comment:
Ultimately, it would be good to have matching rules on the body itself, but I don't think that is possible at this stage.